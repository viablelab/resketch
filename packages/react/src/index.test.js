import * as exported from './';

test('exports', () => {
  expect(exported).toMatchSnapshot();
});
