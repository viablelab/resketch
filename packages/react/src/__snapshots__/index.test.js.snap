// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports 1`] = `
Object {
  "Button": [Function],
  "Checkbox": [Function],
  "Input": [Function],
  "Radio": [Function],
  "Select": [Function],
  "Switch": [Function],
}
`;
